build.sh

```diff
@@ -21,14 +21,16 @@ function build() {
     source ${distro}/interface.sh
     SHASUM=$(containerdisks::needs_update ${tag})
     if [ -n "${SHASUM}" ]; then
-      docker build -t quay.io/containerdisks/${distro}:${tag} -t quay.io/containerdisks/${distro}:${tag}-${TAG_SUFFIX} --label "shasum=${SHASUM}" -f ${distro}/${tag}/Dockerfile .
+      docker build -t quay.io/containerdisks/${distro}:${tag} -t quay.io/containerdisks/${distro}:${tag}-${TAG_SUFFIX} -t latest --label "shasum=${SHASUM}" -f ${distro}/${tag}/Dockerfile .
```

Comment:
@dominikholler I would prefer if we don't introduce `latest`. Are you sure that it is needed?

Reply:
Like I would definitely want to do a conscious move from centos 8 to centos 9 and `8` and `9` are already moving tags.

Reply:
I am fine, so we have to document that the tag is used explicitly, e.g. quay.io/dholler/centos:8 .